doc/identity_hub.md

```diff
@@ -0,0 +1,756 @@
+# Identity Hub integration
+
+Table of content:
+
+  - [Download and customize to run locally](#download)
+  - [Creating participant context](#creating-participant-context)
+  - [Get participants](#get-participants)
+  - [DID Documents](#did-documents)
+  - [Verifiable credentials](#vc)
+  - [Dataspace-issuer did document](#issuer-did-document)
+  - [Further reading](#more)
+
+Note: informations gathered so far:
+
+## Download and customize to run locally <a name="download"></a>
+
+- Download [identity hub](https://github.com/eclipse-edc/IdentityHub/tree/main); (use released versions v0.10.0 or v.0.11.0)
+- Apply following changes (to support super-admin user; removed because security, so far no information how to run and not get 403 on endpoints)
+
+Copy [MinimumViableDataspace-main\extensions\superuser-seed\](https://github.com/eclipse-edc/MinimumViableDataspace/tree/main/extensions/superuser-seed) to IdentityHub-main\extensions\superuser-seed\
+
+```
+gradle.properties
+
+version=0.10.0
+
+```
+
+```
+‎gradle/libs.versions.toml
+
+edc = "0.10.0"
+
+
+edc-ih-spi-store = { module = "org.eclipse.edc:identity-hub-store-spi", version.ref = "edc" }
+
+```
+
+```
+‎‎launcher/identityhub/build.gradle.kts
+
+dependencies {
+    runtimeOnly(project(":dist:bom:identityhub-with-sts-bom"))
+    runtimeOnly(project(":extensions:superuser-seed"))   <-- THIS ONE
+}
+```
+
+
+```
+‎settings.gradle.kts
+
+include(":extensions:superuser-seed")
+
+```
+
+In extensions\superuser-seed\src\main\java\org\eclipse\edc\identityhub\seed\ParticipantContextSeedExtension.java make sure to change line:
+
+```
+   .orElse(generatedKey.apiKey());
+```
+
+with
+
+```
+  .orElse(generatedKey.get("apiKey").toString());
+ 
+```
+Build gradle project using
+
+```
+gradlew :launcher:identityhub:shadowJar
+```
+
+
+When bult, run using following command
+
+```
+java -Dweb.http.presentation.port=10001 -Dweb.http.presentation.path="/api/presentation" -Dweb.http.port=8181 -Dweb.http.path="/api" -Dweb.http.identity.port=8182 -Dweb.http.identity.path="/api/identity" -Dedc.ih.api.superuser.key="c3VwZXItdXNlcg==.c3VwZXItc2VjcmV0Cg==" -Dedc.api.accounts.key="password" -jar launcher/identityhub/build/libs/identity-hub.jar
+
+```
+
+Import postman collection from [MVDS project](https://github.com/eclipse-edc/MinimumViableDataspace/blob/main/deployment/postman/MVD.postman_collection.json) and corresponding [environment file.](https://github.com/eclipse-edc/MinimumViableDataspace/blob/main/deployment/postman/MVD%20Local%20Development.postman_environment.json)
+
+
+## Creating participant context <a name="participant-context"></a>
```

Comment:
anchors don't match #creating-participant-context vs participant-context

Reply:
Fixed in ToC